packages/shared/prisma/schema.prisma

```diff
@@ -726,7 +726,7 @@ model Prompt {
     version  Int
     type     String   @default("text")
     isActive Boolean? @map("is_active") // Deprecated. To be removed once 'production' labels work as expected.
-    config   Json     @default("{}")
+    config   Json     @default("{}") @db.Json
```

Comment:
Prisma tries to revert back to default `JSONB` without the `@db.Json`. See here: https://www.prisma.io/docs/orm/reference/prisma-schema-reference#postgresql-7